src/components/DurationSpan.vue

```diff
@@ -22,37 +23,65 @@ export default {
   watch: {
     endTime() {
       this.updateDuration()
+      this.triggerUpdateDurationIntervalAtNextMinute()
     }
   },
   mounted() {
     this.updateDuration()
+    this.triggerUpdateDurationIntervalAtNextMinute()
   },
   beforeDestroy() {
     clearInterval(this.interval)
   },
   methods: {
     updateDuration() {
+      this.duration = durationDifference(
+        new Date(this.startTime),
+        this.endTime ? new Date(this.endTime) : new Date()
+      )
+    },
+    triggerUpdateDurationIntervalAtNextMinute() {
       clearInterval(this.interval)
-      if (!this.endTime) {
-        let now = new moment(),
-          start = new moment(this.startTime)
-        this.duration = moment.duration(now.diff(start))
+      const millisecondsUntilNextMinute = getMillisecondsUntilNextMinute()
 
-        this.interval = setInterval(() => {
-          let now = new moment(),
-            start = new moment(this.startTime)
-          this.duration = moment.duration(now.diff(start))
-        }, 1000)
-      } else {
-        let end = new moment(this.endTime),
-          start = new moment(this.startTime)
-        this.duration = moment.duration(end.diff(start))
-      }
+      this.interval = setTimeout(
+        this.updateDurationOnMinuteInterval,
+        millisecondsUntilNextMinute
+      )
+    },
+    updateDurationOnMinuteInterval() {
+      this.updateDuration()
+      this.interval = setInterval(this.updateDuration, 60000)
+    },
+    plural(word, count) {
+      return count == 1 ? word : `${word}s`
     }
   }
 }
 </script>
 
 <template>
-  <span>{{ duration | duration }}</span>
+  <span class="duration-span">
+    <span class="duration-span__days"
+      >{{ duration.days }} {{ plural('day', duration.days.length) }}</span
```

Comment:
I think we need to check if `duration` is null for this span, I'm seeing errors locally 

```
Vue Global Error Handler {error: TypeError: Cannot read properties of null (reading 'days')
```